### DISCUSSION TITLE
Empty MAUI project does not compile, because I have installed both Visual Studio (stable) and Visual Studio Preview

### DISCUSSION THREAD

**Discussion by TommiGustafsson-HMP** [dotnet/maui#16566] [Q&A]
I try to compile an empty MAUI project created in Visual Studio 2022 (stable), but it does not compile, since it it does not find workloads. The build says among other things: 

`C:\Program Files\dotnet\sdk\7.0.306\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: android`

When I do `dotnet workload list`, I get:

```
Installed Workload Id      Manifest Version                            Installation Source
-----------------------------------------------------------------------------------------------------------
ios                        16.4.8646-net8-p6/8.0.100-preview.6         VS 17.6.33829.357, VS 17.7.33927.210
maui-windows               8.0.0-preview.6.8686/8.0.100-preview.6      VS 17.6.33829.357, VS 17.7.33927.210
maui-maccatalyst           8.0.0-preview.6.8686/8.0.100-preview.6      VS 17.6.33829.357, VS 17.7.33927.210
maui-ios                   8.0.0-preview.6.8686/8.0.100-preview.6      VS 17.6.33829.357, VS 17.7.33927.210
maccatalyst                16.4.8646-net8-p6/8.0.100-preview.6         VS 17.6.33829.357, VS 17.7.33927.210
android                    34.0.0-preview.6.359/8.0.100-preview.6      VS 17.6.33829.357, VS 17.7.33927.210
maui-android               8.0.0-preview.6.8686/8.0.100-preview.6      VS 17.6.33829.357, VS 17.7.33927.210
```

Indeed, it appears that I do not have 7.0 workloads installed. I think this is because I have both Visual Studio 2022 (stable) and Visual Studio 2022 Preview installed.

How am I supposed to develop a MAUI app using Visual Studio 2022 (stable)? That is, how am I supposed to install MAUI 7.0 workloads side-by-side with 8.0.0-preview6 workloads, which I need for Visual Studio 2022 Preview?

**Comment by TommiGustafsson-HMP** [comment]
It seems that `dotnet build` works on the command line, but building/rebuilding in Visual Studio does not work. Is this a tooling problem?

Edit: Building the project with VS 2022 Preview seems to work fine. But the stable version does not wok.

**Comment by Xyncgas** [comment]
1st, uninstall & reinstall (stable) and (preview)
restart your computer if you are using windows
install the correct workload for MAUI using visual studio's UI
try programming again
if problem still exist I would recommend to contact Microsoft support
